### ISSUE TITLE
1.16.4 Exit Code 0 crash and missing vanilla inventory in creative

### ISSUE THREAD

**Issue by AshenToaster** [FabricMC/fabric#1304]
Recently updated to 1.16.4 with all my usual mods. 

After a certain amount of mods are installed, Minecraft will occasionally crash upon launch with an "Exit Code 0" message without any logs. When successfully loading, the vanilla blocks/items are missing in the creative menu, only the modded items show up, however RoughlyEnoughItems (REI) doesn't have the same issue.

I'm using the Fabric Loader Version 0.11.1 and Fabric API Version 0.29.3

Here's a pastebin with my mods if it helps: https://pastebin.com/NAifNeEA

**Comment by liach** [comment]
Do you have a `debug.log` or `latest.log` available in your logs folder? and do you have crash reports in `crash-reports` folder? Looking at the details generated, there should be some crash report at least.

**Comment by AshenToaster** [comment]
There were are no crash reports. Here's the latest log: https://pastebin.com/if4SVANQ

**Comment by liach** [comment]
Hmm, if there is exit code 0, maybe some mod is calling it. No clue in log either.

**Comment by AshenToaster** [comment]
Tried updating Optifine/Optifabric since it was bringing up errors in the log but no results.

Any idea how to find which mod could be doing this? I don't know enough about how this works to find a solution. I did notice that the crashes and missing errors start happening once I have more than a certain amount of mods installed, like 10 or so, or perhaps it's one of the libraries or dependencies I have that is causing these issues.
